x-pack/plugins/security/server/config_deprecations.ts

```diff
@@ -162,29 +162,47 @@ export const securityConfigDeprecationProvider: ConfigDeprecationProvider = ({
   (settings, fromPath, addDeprecation) => {
     if (settings?.xpack?.security?.session?.idleTimeout === undefined) {
       addDeprecation({
-        message:
-          'Session idle timeout ("xpack.security.session.idleTimeout") will be set to 1 hour by default in the next major version (8.0).',
+        title: i18n.translate('xpack.security.deprecations.idleTimeoutTitle', {
+          defaultMessage: 'Default value of xpack.security.session.idleTimeout is changing',
+        }),
+        message: i18n.translate('xpack.security.deprecations.idleTimeoutMessage', {
+          defaultMessage:
+            'Session idle timeout ("xpack.security.session.idleTimeout") will be set to 1 hour by default in the next major version (8.0).',
+        }),
         documentationUrl:
           'https://www.elastic.co/guide/en/kibana/current/xpack-security-session-management.html#session-idle-timeout',
         correctiveActions: {
           manualSteps: [
-            `Use "xpack.security.session.idleTimeout" in your Kibana configuration to change default session idle timeout.`,
-            `To disable session idle timeout, set "xpack.security.session.idleTimeout" to 0.`,
+            i18n.translate('xpack.security.deprecations.idleTimeout.manualStepOneMessage', {
+              defaultMessage: `Use "xpack.security.session.idleTimeout" in your Kibana configuration to change default session idle timeout.`,
+            }),
+            i18n.translate('xpack.security.deprecations.idleTimeout.manualStepTwoMessage', {
+              defaultMessage: `To disable session idle timeout, set "xpack.security.session.idleTimeout" to 0.`,
+            }),
           ],
         },
       });
     }
 
     if (settings?.xpack?.security?.session?.lifespan === undefined) {
       addDeprecation({
-        message:
-          'Session lifespan ("xpack.security.session.lifespan") will be set to 30 days by default in the next major version (8.0).',
+        title: i18n.translate('xpack.security.deprecations.lifespanTitle', {
+          defaultMessage: 'Default value of xpack.security.session.lifespan is changing',
+        }),
+        message: i18n.translate('xpack.security.deprecations.lifespanMessage', {
+          defaultMessage:
+            'Session lifespan ("xpack.security.session.lifespan") will be set to 30 days by default in the next major version (8.0).',
```

Comment:
The session idle timeout will default to 1 hour in 8.0.
```suggestion
            'The session lifespan will default to 30 days  in 8.0.',
```